UnitsNet.Tests/CustomCode/TemperatureTests.cs

```diff
@@ -63,7 +64,8 @@ public void MultiplyByTemperatureDeltaEqualsTemperature(TemperatureUnit unit, in
             // Act
             Temperature resultTemp = temperature.Multiply(factor, unit);
 
-            string actual = resultTemp.ToUnit(unit).ToString(CultureInfo.InvariantCulture, "{0:0} {1}");
+            var resultTempInUnit = resultTemp.ToUnit(unit);
```

Comment:
Minor, but I think it read slightly better as a single line. Same below.

Reply:
No sense in using strings to check values. I've converted these tests to MemberData driven tests to avoid the issue entirely.